specifyweb/frontend/js_src/lib/components/QueryBuilder/ResultsTable.tsx

```diff
@@ -169,16 +174,48 @@ function Row({
             className={`${getCellClassName(condenseQueryResults)} sticky`}
             role="cell"
           >
-            <Link.NewTab
-              className="print:hidden"
-              href={viewUrl}
-              rel="noreferrer"
-            />
+            {splitRecords === undefined ? (
+              <Link.NewTab
+                className="print:hidden"
+                href={viewUrl}
+                rel="noreferrer"
+              />
+            ) : (
+              <Button.Icon
+                className="print:hidden"
+                icon="viewList"
+                title={queryText.viewRecords()}
+                onClick={() => toggleIsListOfRecordsOpen(true)}
+              />
+            )}
+            {isListOfRecordsOpen && splitRecords !== undefined ? (
+              <RecordSelectorFromIds
+                defaultIndex={0}
+                dialog="modal"
+                headerButtons={undefined}
+                ids={splitRecords}
+                isDependent={false}
+                isInRecordSet={false}
+                newResource={undefined}
+                table={table}
+                title={commonText.colonLine({
+                  label: queryText.queryResults(),
+                  value: table.label,
+                })}
+                totalCount={splitRecords.length}
+                onAdd={undefined}
+                onClone={undefined}
+                onClose={() => toggleIsListOfRecordsOpen(false)}
+                onDelete={undefined}
```

Comment:
I am fine with delete not being supported, here, but just wanted to mention that it won't be very complicated:

you store `splitRecords` in a state
onDelete you remove one item from that state
you could also look at how "Browse in forms" does it right now

Reply:
Yes, I had a delete option before but we decided to not support it here.